modules/ROOT/pages/sftp/sftp-list.adoc

```diff
@@ -80,28 +78,43 @@ minSize="0"
-
-Individual attributes:
-
-* accessedSince: Inclusive lower boundary for the file access stamp expressed as either a DateTime instance or a String in ISO-8601 format.
-* accessedUntil: Inclusive upper boundary for the file access stamp expressed as either a DateTime instance or a String in ISO-8601 format.
-* createdSince: Inclusive lower boundary for the file creation stamp expressed as either a DateTime instance or a String in ISO-8601 format.
-* createdUntil: Inclusive upper boundary for the file creation stamp expressed as either a DateTime instance or a String in ISO-8601 format.
-* directory: Match only if the file is a directory.
-* filenamePattern: Similar to the current filename pattern filter but more powerful. Glob expressions and regex are supported. Glob is the default. You can select which one to use by setting a prefix. For example: `glob:*.{java, js}` , regex:`[0-9]_test.csv`
-* maxSize: Inclusive upper boundary for the file size expressed in bytes.
-* minSize: Inclusive lower boundary for the file size expressed in bytes.
-* path-pattern: Same as filenamePattern but applies over the entire file path. Not just a filename.
-* regularFile: Match only if the file is a regular file.
-* symbolicLink: Match only if the file is a symbolic link.
-* updatedSince: Inclusive lower boundary for the file modification stamp expressed as either a DateTime instance or a String in ISO-8601 format.
-* updatedUntil: Inclusive upper boundary for the file modification stamp expressed as either a DateTime instance or a String in ISO-8601 format.
+=== Individual Attributes
+
+Individual attributes are optional and are ignored if not provided.  You can use an `AND` operator to combine individual attributes. 
+
+* `accessedSince` +
+Inclusive lower boundary for the file access stamp expressed as either a DateTime instance or a String in ISO-8601 format.
+* `accessedUntil` + 
+Inclusive upper boundary for the file access stamp expressed as either a DateTime instance or a String in ISO-8601 format.
+* `createdSince` +
+Inclusive lower boundary for the file creation stamp expressed as either a DateTime instance or a String in ISO-8601 format.
+* `createdUntil` +
+Inclusive upper boundary for the file creation stamp expressed as either a DateTime instance or a String in ISO-8601 format.
+* `directory` +
+Match only if the file is a directory.
+* `filenamePattern` +
+Similar to the current filename pattern filter but more powerful. Glob expressions (default) and regex are supported. You can select which one to use by setting a prefix. For example: `glob:*.{java, js}` or `regex:[0-9]test.csv`. 
+* `maxSize` +
+Inclusive upper boundary for the file size expressed in bytes.
+* `minSize` +
+Inclusive lower boundary for the file size expressed in bytes.
+* `path-pattern` + 
+Same as filenamePattern but applies over the entire file path. Not just a filename.
```

Comment:
Doublechecking the second sentence: are we saying that filenamePattern is not just a filename or that filenamePattern applies over an entire file path and not just a file?